uwblueprint/childrens-aid-society
# [DO NOT MERGE] update packages to resolve firebase hosting errors

## Notion ticket link
no ticket



## Implementation description
* updated packages in frontend to try and resolve firebase hosting error



## Steps to test
1. can pull this, run `docker-compose up --build`, and then verify that the app still functions as expected



## What should reviewers focus on?
* should run with --build flag next time you spin up docker -> `docker-compose up --build`


## Checklist
- [] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR
